src/DssSpell.sol

```diff
@@ -40,52 +40,170 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkAutoLineParameters("GUNIV3DAIUSDC2-A", 10 * MILLION, 10 * MILLION, 8 hours);
+
+        // ----------------------------- Rates updates -----------------------------
+        // Increase the ETH-A Stability Fee from 2.5% to 2.75%
+        DssExecLib.setIlkStabilityFee("ETH-A", TWO_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase the ETH-B Stability Fee from 6.0% to 6.5%
+        DssExecLib.setIlkStabilityFee("ETH-B", SIX_FIVE_PCT_RATE, true);
+
+        // Increase the LINK-A Stability Fee from 1.5% to 2.5%
+        DssExecLib.setIlkStabilityFee("LINK-A", TWO_FIVE_PCT_RATE, true);
+
+        // Increase the MANA-A Stability Fee from 3.0% to 6.0%
+        DssExecLib.setIlkStabilityFee("MANA-A", SIX_PCT_RATE, true);
+
+        // Increase the UNI-A Stability Fee from 1.0% to 3.0%
+        DssExecLib.setIlkStabilityFee("UNI-A", THREE_PCT_RATE, true);
+
+        // Increase the GUSD-A Stability Fee from 0.0% to 1.0%
+        DssExecLib.setIlkStabilityFee("GUSD-A", ONE_PCT_RATE, true);
+
+        // Increase the UNIV2DAIETH-A Stability Fee from 1.5% to 2.0%
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", TWO_PCT_RATE, true);
+
+        // Increase the UNIV2WBTCETH-A Stability Fee from 2.5% to 3.0%
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", THREE_PCT_RATE, true);
+
+        // Increase the UNIV2USDCETH-A Stability Fee from 2.0% to 2.5%
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", TWO_FIVE_PCT_RATE, true);
+
+        // Increase the UNIV2UNIETH-A Stability Fee from 2.0% to 4.0%
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FOUR_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC1-A Stability Fee from 0.5% to 0.1%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_ONE_PCT_RATE, true);
+
+        // ----------------------------- Debt Ceiling updates -----------------------------
+        // Increase the WBTC-A Maximum Debt Ceiling (line) from 1.5 billion DAI to 2 billion DAI
+        // Increase the WBTC-A Target Available Debt (gap) from 60 million DAI to 80 million DAI
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", 2 * BILLION, 80 * MILLION, 6 hours);
+
+        // Increase the Dust Parameter from 30,000 DAI to 40,000 DAI for the ETH-B
+        DssExecLib.setIlkMinVaultAmount("ETH-B", 40_000);
+
+        // Increase the Dust Parameter from 10,000 DAI to 15,000 DAI for all vault-types excluding ETH-B and ETH-C
+        DssExecLib.setIlkMinVaultAmount("ETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("TUSD-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-C", 15_000);
+        DssExecLib.setIlkMinVaultAmount("KNC-A", 15_000);
```

Comment:
This collatéral is being offboarded
```suggestion

```

Reply:
After discussing with Prose11 in the new-spell discord channel, I think we should leave this increase in. Its seems that was the intent of governance